packages/react-component-ref/src/toRefObject.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+
+const nullRefObject: React.RefObject<null> = { current: null }
```

Comment:
What if the user uses reference like this in the following way: ref.current.focus()? How is it safe to have a `nullRefObject`?

Reply:
It can be `null` by typings:
```tsx
    interface RefObject<T> {
        readonly current: T | null;
    }
    // Mutable it used in hooks an initial value is obviously defined
    // React.useRef(null)
    interface MutableRefObject<T> {
        current: T;
    }
```